### DISCUSSION TITLE
20241031-vpn-history/

### DISCUSSION THREAD

**Discussion by giscus** [inflearn/tech-blog-comment#59] [Comments]
# 20241031-vpn-history/

VPN 솔루션 선택을 위해 고민한 과정과 고려했던 요소를 공유합니다.

https://tech.inflab.com/20241031-vpn-history/

**Comment by yureutaejin** [comment]
아키텍처에 vpn mesh가 포함되어야하면 tailscale이 좋지만, 개인 사용자의 입장에선 nordvpn의 meshnet도 세팅하기 무척 편해서 좋은 것 같습니다. :)

**Comment by iamb0ttle** [comment]
유익한 정보 알아갑니다 👍
